examples/rdkafka_performance.c

```diff
@@ -1432,9 +1436,6 @@ int main (int argc, char **argv) {
 		outq = rd_kafka_outq_len(rk);
                 if (verbosity >= 2)
                         printf("%% %i messages in outq\n", outq);
-		cnt.msgs -= outq;
```

Comment:
These should not be removed, they're needed for proper final stats.

Reply:
have done it